django_select2/static/django_select2/django_select2.js

```diff
@@ -1,4 +1,4 @@
-$(function () {
+django.jQuery(function($) {
```

Comment:
Well first of all, *never* rely on the ``$`` symbol at a global scope. Good practice for all jQuery modules is to start with something such as

```
jQuery(function($) { 
   ...
);
```

or

```
(function($) {
    ...
})(jQuery);
```

this reduces the scope of the ``$`` symbol into that function and keeps the global scope clean.
Every portable jQuery library I know uses one of those patterns (or similar) and does not reply on a global ``$``.

See here: http://code.tutsplus.com/tutorials/jquery-anti-patterns-and-best-practices--pre-45600

Now its debatable whether to use ``jQuery`` or ``django.jQuery``. The Django docs state:
https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#jquery

If django-select2 is used in Django's admin backend, then ``django.jQuery`` is already defined and shall be used instead of a second include. Otherwise I agree, that the pattern ``jQuery(function($) { ...`` makes more sense.